format/bplist/testdata/bplist.fqtest

```diff
@@ -0,0 +1,2269 @@
+$ fq -d bplist dv com.apple.UIAutomation.plist
+    |00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f|0123456789abcdef|.{}: com.apple.UIAutomation.plist (bplist) 0x0-0x44.7 (69)
+    |                                               |                |  header{}: 0x0-0x7.7 (8)
+0x00|62 70 6c 69 73 74                              |bplist          |    magic: "bplist" (valid) 0x0-0x5.7 (6)
+0x00|                  30 30                        |      00        |    version: "00" (valid) 0x6-0x7.7 (2)
+    |                                               |                |  objects{}: 0x8-0x21.7 (26)
+0x00|                        d1                     |        .       |    type: "dict" (13) (Dictionary) 0x8-0x8.3 (0.4)
+0x00|                        d1                     |        .       |    size_bits: 1 0x8.4-0x8.7 (0.4)
+    |                                               |                |    size: 1 0x9-NA (0)
+    |                                               |                |    entries[0:1]: 0x9-0x21.7 (25)
+    |                                               |                |      [0]{}: entry 0x9-0x21.7 (25)
+0x00|                           01                  |         .      |        key_index: 1 0x9-0x9.7 (1)
+0x00|                              02               |          .     |        value_index: 2 0xa-0xa.7 (1)
+    |                                               |                |        key{}: 0xb-0x20.7 (22)
+0x00|                                 5f            |           _    |          type: "ascii_string" (5) (ASCII encoded string) 0xb-0xb.3 (0.4)
+0x00|                                 5f            |           _    |          size_bits: 15 0xb.4-0xb.7 (0.4)
+0x00|                                    10         |            .   |          large_size_marker: 1 (valid) 0xc-0xc.3 (0.4)
+0x00|                                    10         |            .   |          exponent: 0 0xc.4-0xc.7 (0.4)
+0x00|                                       13      |             .  |          size_bigint: 19 0xd-0xd.7 (1)
+    |                                               |                |          size: 19 0xe-NA (0)
+0x00|                                          55 49|              UI|          value: "UIAutomationEnabled" 0xe-0x20.7 (19)
+0x10|41 75 74 6f 6d 61 74 69 6f 6e 45 6e 61 62 6c 65|AutomationEnable|
+0x20|64                                             |d               |
+    |                                               |                |        value{}: 0x21-0x21.7 (1)
+0x20|   09                                          | .              |          type: "singleton" (0) (Singleton value (null/bool)) 0x21-0x21.3 (0.4)
+    |                                               |                |          value: true 0x22-NA (0)
+0x20|   09                                          | .              |  unknown0: raw bits 0x21.4-0x21.7 (0.4)
```

Comment:
I think these are the unused 4 bit when size is < 0xf? maybe add a "unused" field for them etc?

Reply:
I'm not seeing this in the output anymore, so I think it is fixed? Can't remember right now if I explicitly fixed this, let me know if not.